rdr_service/tools/tool_libs/questionnaire_response_duplicate_cleanup.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import sys
+import pytz
+
+from calendar import monthrange
+from datetime import datetime, timedelta
+from rdr_service.services.response_duplication_detector import ResponseDuplicationDetector
+
+_logger = logging.getLogger("rdr_logger")
+
+# Tool_cmd and tool_desc name are required.
+# Remember to add/update bash completion in 'tool_lib/tools.bash'
+tool_cmd = "qr_duplicate_fix"
+tool_desc = "Tool to run the questionnaire response duplicate detection logic over a long period of time."
+
+
+def run():
+
+    # Starting from year of oldest observed duplicates
+    start = datetime(2017, 1, 31, 23, 59, 59, 999999, pytz.UTC)
+    rdd = ResponseDuplicationDetector()
```

Comment:
I'm not familiar with `ResponseDuplicationDetector()`. How is the class aware of which GCP project to flag the records in?
Using the other tools in tool_libs as a template, you can see how arguments are passed to the tool.

Reply:
The class does have a default, which I realize looking close will use the run location so localhost in my case. I will add the logic to take a project param and use that, thanks for catching.

Reply:
Also can you make sure this inherits from the ToolBase class to fit the same pattern as the rest of the tools?